suite/tests/CMakeLists.txt

```diff
@@ -2647,6 +2647,12 @@ if (NOT ANDROID) # TODO i#38: Port test to Android.
       client-interface/attach_blocking.runall "" "" "")
   endif ()
   if (X64 AND (LINUX OR WIN32))
+    if (LINUX)
+      set(client.memory_dump_test_runcmp "${CMAKE_CURRENT_SOURCE_DIR}/runmulti.cmake")
+      set(client.memory_dump_test_precmd "rm@${PROJECT_BINARY_DIR}/logs/simple_app.*.elf")
+      set(client.memory_dump_test_postcmd "readelf@-a@${PROJECT_BINARY_DIR}/logs/simple_app.*.elf")
```

Comment:
The readelf binary needs to be located: may not be there; may not be on path.

Xref `find_program(READELF_EXECUTABLE` in core/.

Reply:
Thanks for the pointer, I've replaced readelf with ${READELF_EXECUTABLE}.